libcloud/storage/types.py

```diff
@@ -106,6 +106,10 @@ class Provider(object):
     S3_RGW = "s3_rgw"
     S3_RGW_OUTSCALE = "s3_rgw_outscale"
     MINIO = "minio"
+    SCALEWAY = "scaleway"
+    SCALEWAY_FR_PAR = "scaleway_fr_par"
```

Comment:
Please remove region specific constants and leave just ``SCALEWAY`` here - as discussed, we moved away from region specific constants and old ones are here for backward compatibility reasons.

Reply:
It slipped away...